sdk/storage/storage-blob/src/Clients.ts

```diff
@@ -3167,15 +3167,7 @@ export interface BlobQueryError {
  * @export
  * @interface BlobQueryTextConfiguration
  */
-export interface BlobQueryTextConfiguration {
-  /**
-   * Record separator.
-   *
-   * @type {string}
-   * @memberof BlobQueryTextConfiguration
-   */
-  recordSeparator: string;
-}
+export interface BlobQueryTextConfiguration {}
```

Comment:
>BlobQueryTextConfiguration [](start = 17, length = 26)

"Move separator to child class for future feature". What's the future feature here?

Reply:
For some arrow type which has no `recordSeparator` field. Empty base type removed.